video/mp_image.c

```diff
@@ -1168,11 +1168,7 @@ struct AVFrame *mp_image_to_av_frame(struct mp_image *src)
     if (src->fields & MP_IMGFIELD_REPEAT_FIRST)
         dst->repeat_pict = 1;
 
-    dst->colorspace = pl_system_to_av(src->params.repr.sys);
-    dst->color_range = pl_levels_to_av(src->params.repr.levels);
-    dst->color_primaries =
-        pl_primaries_to_av(src->params.color.primaries);
-    dst->color_trc = pl_transfer_to_av(src->params.color.transfer);
+    pl_avframe_set_repr(dst, src->params.repr);
```

Comment:
I think we need `_color` here too

Reply:
That's alreasy called lower down in the function.

Reply:
https://github.com/mpv-player/mpv/blob/a45518cf571fd1b43482aa4720eeb7a4b0b1c684/video/mp_image.c#L1191

Reply:
If possible please move them closer together, they are linked and is nicer to have them both next to each other